frontend/src/utils/constants.ts

```diff
@@ -2,7 +2,7 @@ export const BucketConfig = Object.freeze({
   HEADER_NEW_BUCKET: 'Configure bucket',
   TITLE_NEW_BUCKET: 'Use this form to configure a bucket to be used in BCBox for the first time.',
   NOTICE_NEW_BUCKET: 'If you intend to share files in your bucket with BCeID users, you are required'
-  + ' to email IDIM.Consulting@gov.bc.ca to share your intentions and where you intend to advertise this.'
+    + ' to email IDIM.Consulting@gov.bc.ca to share your intentions and where you intend to advertise this.'
```

Comment:
At some point we're going to need to revisit this and have this be driven from the /config endpoint. We should put in a TODO comment here so that we don't forget to address this down the line.